puppet/modules/quickstack/manifests/params.pp

```diff
@@ -58,9 +58,9 @@
   $cinder_dell_sc_san_login        = ['Admin'],
   $cinder_dell_sc_san_password     = ['CHANGEME'],
   $cinder_dell_sc_iscsi_ip_address = ['192.168.0.20'],
-  $cinder_dell_sc_iscsi_port       = [3260],
+  $cinder_dell_sc_iscsi_port       = ['3260'],
```

Comment:
They have not been removed for legacy reasons, but no, we are not doing any new development on the listed files.